src/components/app-bar/BurgerMenu.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+	import {
+		activeAccount,
+		activeAccountBounties,
+		walletConnect as wcConnection
+	} from '../../stores';
+	import { onMount } from 'svelte';
+
+	let open = false;
+	let container: HTMLDivElement | null = null;
+
+	function toggleBurgerMenu() {
+		open = !open;
+	}
+
+	$: {
+		if (open) {
+			window.addEventListener('click', closeMenuOnOutsideClick);
+		} else {
+			window.removeEventListener('click', closeMenuOnOutsideClick);
+		}
+	}
+
+	async function logOut() {
+		activeAccount.set(undefined);
+		sessionStorage.clear();
+		activeAccountBounties.set([]);
+		if ($wcConnection) {
+			await $wcConnection.disconnect();
+		}
+		closeBurgerMenu();
+	}
+
+	function closeBurgerMenu() {
+		open = false;
+	}
+
+	function closeMenuOnOutsideClick(event: MouseEvent) {
+		if (!container?.contains(event.target as Node)) {
+			closeBurgerMenu();
+		}
+	}
+
+	onMount(() => {
+		return () => {
+			window.removeEventListener('click', closeMenuOnOutsideClick);
+		};
+	});
+</script>
+
+<div bind:this={container} class="relative">
+	<!-- Burger Menu Icon -->
+	<button on:click={toggleBurgerMenu}>
+		<span class="material-symbols-rounded text-white"> menu </span>
+	</button>
+
+	<!-- Menu Overlay -->
+	{#if open}
+		<div
+			class="absolute top-0 right-0 flex flex-col items-start bg-backgroundContent border shadow-lg p-2.5 rounded-md w-[210px]"
+		>
+			<button on:click={toggleBurgerMenu} class="self-end mr-2.5 mb-1">
```

Comment:
```suggestion
		<div
		  onclick={() => { open = false; }}
			class="absolute top-0 right-0 flex flex-col items-start bg-backgroundContent border shadow-lg p-2.5 rounded-md w-[210px]"
		>
			<button class="self-end mr-2.5 mb-1">
```
I can’t see the design now, but I see that clicking any item in the menu results in closing the menu. Could we just have a single even handler on the menu itself? Or are there menu areas which should be clickable without closing the menu?